archunit/src/jdk9test/java/com/tngtech/archunit/core/importer/LambdaExampleTest.java

```diff
@@ -0,0 +1,39 @@
+package com.tngtech.archunit.core.importer;
+
+import java.util.function.Supplier;
+
+
+public class LambdaExampleTest {
```

Comment:
Did you want to commit this? It looks like it's just a little class to experiment? Since I don't see any assertions that would actually "test" anything?